cmd/utils/flags.go

```diff
@@ -1891,6 +1899,27 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 		if !ctx.IsSet(MinerGasPriceFlag.Name) {
 			cfg.Miner.GasPrice = big.NewInt(1)
 		}
+	case ctx.IsSet(ChainFlag.Name):
```

Comment:
We should probably add this flag to `CheckExclusive` at the top of `SetEthConfig`